src/main/io/ledstrip.c

```diff
@@ -741,6 +742,16 @@ void updateLedStrip(void)
         return;
     }
 
+    if ( IS_RC_MODE_ACTIVE(BOXLEDLOW)){
+        if (ledStripIsOnFlag){
+    		ledStripDisable();
+            ledStripIsOnFlag = false;
+        }
+        return;
+    }else{
+        ledStripIsOnFlag = true;
+    }
+
```

Comment:
changed name to ledStripEnabled... makes sense.

added a check on ledStripIsOnFlag... it does simplify the flow  :)